src/pkg/packager/compose.go

```diff
@@ -203,6 +203,7 @@ func (p *Packager) mergeComponentOverrides(target *types.ZarfComponent, override
 	target.Images = append(target.Images, override.Images...)
 	target.Manifests = append(target.Manifests, override.Manifests...)
 	target.Repos = append(target.Repos, override.Repos...)
+	target.Extensions.BigBang.ValuesFiles = append(target.Extensions.BigBang.ValuesFiles, override.Extensions.BigBang.ValuesFiles...)
```

Comment:
ValuesFiles may not be instantiated (hence the nil pointer error) - also it may be better to implement a compose interface for extensions to implement (something like):

```
type Extension interface {
	Compose() interface{}
}
```

That way there is a pattern for future extensions to implement that isn't as coupled to Zarf code.